Source/Blazorise/wwwroot/breakpoint.js

```diff
@@ -2,15 +2,17 @@
 const breakpointComponents = [];
 let lastBreakpoint = null;
 
-// Recalculate breakpoint on resize
-if (window.attachEvent) {
-    window.attachEvent('onresize', windowResized);
-}
-else if (window.addEventListener) {
-    window.addEventListener('resize', windowResized, true);
-}
-else {
-    //The browser does not support Javascript event binding
+if (document.readyState === 'interactive' || document.readyState === 'complete') {
```

Comment:
I remember using the event DOMContentLoaded for doing stuff when the Dom is available. Is that no better? Since it seems like you are not guaranting that this code attaches the events in case the js is loaded and the document is not on those states? Am I wrong?

Reply:
The JS module deferred loaded, so we cannot listen for DOMContentLoaded.

Reply:
Sorry I don't follow. Do you mean, that tipically it will execute after the DOMContentLoaded has already been called?

How do you guarantee if the following condition is not true `(document.readyState === 'interactive' || document.readyState === 'complete')` that the events will be registered?